### DISCUSSION TITLE
Failed to reset password. {"status":"FORBIDDEN","message":"Resource 'admin' is reserved."}

### DISCUSSION THREAD

**Discussion by janowo2008** [wazuh/wazuh#25123] [Q&A]
After successful installation  I can't change Wazuh-DASHBOARD  @password.
![4_Zrzut ekranu 2024-08-09 150657](https://github.com/user-attachments/assets/14d54fe7-a7fe-4f3e-bdb7-bc24fc295423)

How I can change Wazuh-DASHBOARD password in another way ?

**Comment by Damian-Mangold** [comment]
Hi @janowo2008, Please note that the admin password must be changed via the command line, as it is not possible to modify it through the GUI. Below are some resources that provide detailed steps for updating the password:

- [Wazuh Documentation - Password Management](https://documentation.wazuh.com/current/user-manual/user-administration/password-management.html)
- [Wazuh Google Groups - Admin Password Reset](https://groups.google.com/g/wazuh/c/G9A_Kb7ERQA)
- [Medium Article - Wazuh Part 4: Admin Password Reset](https://medium.com/@bl77rd/wazuh-part-4-admin-password-reset-a095c28de77b)

Please refer to these links for comprehensive instructions on how to proceed with changing the admin password.

**Reply by bbbowles** [reply]
Good morning!

I'm currently facing the same problem on a Docker-hosted Wazuh system, and I keep asking myself: why does the admin user even have the option to try and change the password through the GUI at all? Furthermore, if this is an expected error, why not give the user a detailed error message instead of a FORBIDDEN error?

Other than that, even by following the referenced links and using the password reset tools inside the indexer container, I encountered multiple errors:

The script won't run if the sudo package is not manually installed, which forces you to use the root user to install the package first.

Even with sudo, you must configure the wazuh-indexer user to not ask for its password when invoking the sudo command, as it does not have a password (wazuh-indexer:!!:20697::::::, cue the !! part).

Even after all this trouble, the command does not find an admin user, leading to the error:

    "...ERROR: The given user does not exist"
    
DEPLOYMENT INFORMATION:
Wazuh Version 5.0.0-beta5 on all 3 containers. Although this problem was encoutered on version 4.x(Dont know the full version, sorry) by a colleague months ago.

**Comment by sebasfalcone** [comment]
Inactive thread, please re-open it if necessary
